dist/s_clang_scan

```diff
@@ -34,6 +34,9 @@ $scan $args cmake -G Ninja ../. > /dev/null
 $scan $args ninja wt > ../$t_out 2>&1
 cd ..
 
+# Strip comments from the diff file.
+sed -i '/^#/d' dist/s_clang_scan.diff
```

Comment:
This is modifying in-place and on each run of `s_clang_scan` we're left with this file stripped of comments. We'll need to save the stripped output to a temp file for comparisons.

Reply:
[c92ff7c](https://github.com/wiredtiger/wiredtiger/pull/10481/commits/c92ff7c294f63b1248f4d5a2ca48ebd9541bf861)